src/erlcloud_lambda.erl

```diff
@@ -215,6 +216,34 @@ delete_event_source_mapping(Uuid, Config) ->
     Path = base_path() ++ "event-source-mappings/" ++ binary_to_list(Uuid),
     lambda_request(Config, delete, Path, undefined).
 
+%%------------------------------------------------------------------------------
+%% DeleteFunction
+%%------------------------------------------------------------------------------
+
+%%------------------------------------------------------------------------------
+%% @doc
+%% Lambda API:
+%% [https://docs.aws.amazon.com/lambda/latest/dg/API_DeleteFunction.html]
+%%
+%% ===Example===
+%%
+%%------------------------------------------------------------------------------
+-spec delete_function(FunctionName :: binary()) -> return_val().
+delete_function(FunctionName) ->
+    delete_function(FunctionName, default_config()).
+
+-spec delete_function(FunctionName :: binary(),
+    Config       :: aws_config()) -> return_val().
+delete_function(FunctionName, Config) ->
+    delete_function(FunctionName, undefined, Config).
+
+-spec delete_function(FunctionName :: binary(),
+    Qualifier    :: undefined | binary(),
+    Config       :: aws_config()) -> return_val().
+delete_function(FunctionName, Qualifier, Config) ->
+    Path = base_path() ++ "functions/" ++ binary_to_list(FunctionName),
```

Comment:
Please use [`url_parameter(FunctionName)`](https://github.com/erlcloud/erlcloud/pull/691/files#diff-cc55208044e278022117428643ef16dba42d20c462b93fe80c5447dab2d84309R110) here.

Reply:
Rebased and fixed